### PR TITLE
adjust some tests for cant dodge fee

### PR DESCRIPTION
**Pull request by nhtyy**

### CODE REVIEW

File: contracts/test/core/invoke/Issuance.t.sol [hunk 1]
```diff
@@ -7,39 +7,40 @@ import {SCALAR, fmul} from "src/lib/FixedPoint.sol";
 import {IVault} from "src/interfaces/IVault.sol";
 
 contract IssuanceTest is StatefulTest {
-    function testGoToZero() public {
```
Comment (nhtyy) [inline contracts/test/core/invoke/Issuance.t.sol:10]: commented these out for now because it wont go to zero because fees are minted before burn

**Comment by junhohong** [comment]
Closing for the time being while #20 is in review

File: contracts/test/core/invoke/Issuance.t.sol (hunk 1 above)
Comment (junhohong) [reply contracts/test/core/invoke/Issuance.t.sol:10]: the "go to zero" is actually misleading here, these tests actually now check whether there's dust left over or not.

File: contracts/test/core/invoke/Issuance.t.sol [hunk 2]
```diff
@@ -148,21 +149,21 @@ contract IssuanceTest is StatefulTest {
...
 
         // Redeem tokens
         burn(2e18);
 
-        for (uint256 i; i < virtualUnits.length; i++) {
-            assertEq(
-                IERC20(virtualUnits[i].token).balanceOf(address(this)),
-                startingBalances[i] + (virtualUnits[i].units * 2e18) / SCALAR
+        for (uint256 i; i < units.length; i++) {
+            requireCloseX17(
```
Comment (junhohong) [inline contracts/test/core/invoke/Issuance.t.sol:164]: why the x17 and not eq?

**Pull request closed without merging**